monero-outreach-docs/translations/es/PSA/files/learning/learning_es.sla

```diff
@@ -64,15 +68,59 @@
+        <PAGEOBJECT OwnPage="0" PTYPE="4" XPOS="136" YPOS="380" WIDTH="255.6" HEIGHT="300" RADRECT="0" FRTYPE="0" CLIPEDIT="0" PWIDTH="1" PCOLOR="None" PCOLOR2="None" TXTFILLSH="90" COLUMNS="1" COLGAP="0" NAMEDLST="" SHADE="100" SHADE2="100" GRTYP="0" ROT="0" PLINEART="1" PLINEEND="0" PLINEJOIN="0" LINESP="13.2" LOCALSCX="1" LOCALSCY="1" LOCALX="0" LOCALY="0" PICART="1" PLTSHOW="0" BASEOF="0" textPathType="0" textPathFlipped="0" FLIPPEDH="0" FLIPPEDV="0" IFONT="Adobe Caslon Pro Regular" ISIZE="11" SCALETYPE="1" RATIO="1" PRINTABLE="1" ANNOTATION="0" ANNAME="" TEXTFLOWMODE="0" TEXTFLOW="0" TEXTFLOW2="0" TEXTFLOW3="0" AUTOTEXT="0" EXTRA="0" TEXTRA="0" BEXTRA="0" REXTRA="0" FLOP="0" PFILE="" PFILE2="" PFILE3="" PRFILE="" EPROF="" IRENDER="1" EMBEDDED="1" LOCK="0" LOCKR="0" REVERS="0" TransValue="0" TransValueS="0" TransBlend="0" TransBlendS="0" isTableItem="0" TopLine="0" LeftLine="0" RightLine="0" BottomLine="0" isGroupControl="0" NUMDASH="0" DASHS="" DASHOFF="0" NUMPO="16" POCOOR="0 0 0 0 255.6 0 255.6 0 255.6 0 255.6 0 255.6 300 255.6 300 255.6 300 255.6 300 0 300 0 300 0 300 0 300 0 0 0 0 " NUMCO="16" COCOOR="0 0 0 0 255.6 0 255.6 0 255.6 0 255.6 0 255.6 300 255.6 300 255.6 300 255.6 300 0 300 0 300 0 300 0 300 0 0 0 0 " NUMGROUP="0" GROUPS="" startArrowIndex="0" endArrowIndex="0" OnMasterPage="" ImageClip="" ImageRes="1" Pagenumber="0" isInline="0" fillRule="1" doOverprint="0" gXpos="0" gYpos="0" gWidth="0" gHeight="0" LAYER="0" BOOKMARK="0" NEXTITEM="-1" BACKITEM="-1">
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="A partir del momento de escribir este documento, se ha asumido que la mayoría de la comunidad de Monero consisten en adoptantes primerizos con mayores conocimientos técnicos que una persona del común. "/>
+            <para LINESP="13.2"/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="El código de Monero ha sido trabajado por cientos "/>
+            <para LINESP="13.2"/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="de desarrolladores, que es mucho más de lo que otros proyectos crypto pudiesen aclamar. Algunos especulan que una de las razones del porque, el precio de Monero no ha aumentado al mismo nivel que él de Bitcoin, se debe a que la tecnología detrás de Monero es difícil de comprender y apreciar para la persona del común. Pero nosotros, los primeros adoptantes, seguimos ayudando con la esperanza de que Monero sea reconocida y adoptada ampliamente. Dentro de 100 años, Monero seguirá dependiendo de las personas, personas como "/>
+            <para LINESP="13.2"/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="tú, trabajando para proteger el ecosistema y ayudando "/>
+            <para LINESP="13.2"/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="a que el código avance. Esta es la razón del porque recomendamos que las personas busquen aprender "/>
```

Comment:
"**Estas son las razones por las cuales recomendamos**...". I think it have more meaning than the original line.

Reply:
done